src/Microsoft.OData.Core/Metadata/EdmLibraryExtensions.cs

```diff
@@ -263,6 +277,19 @@ internal static IEnumerable<IEdmOperation> FilterBoundOperationsWithSameTypeHier
         }
 
         /// <summary>
+        /// Filters the operations by parameter names.
+        /// </summary>
+        /// <param name="functions">The operations.</param>
+        /// <param name="parameters">The list of non-binding parameter names to match.</param>
+        /// <param name="caseInsensitive">Whether case insensitive.</param>
+        /// <returns>The best matching operations based on parameters.</returns>
+        internal static IEnumerable<IEdmOperation> FindBestOverloadBasedOnParameters(this IEnumerable<IEdmOperation> functions, IEnumerable<string> parameters, bool caseInsensitive = false)
+        {
+            IEnumerable<IEdmOperation> exactMatches = functions.Where(f => f.Parameters.Count() == parameters.Count() + (f.IsBound ? 1 : 0));

```

Comment:
Yes.  If it is bound then the first parameter is the "binding parameter" and is not used for matching parameters passed in to the function.

---
In reply to: [123296493](https://github.com/OData/odata.net/pull/828#discussion_r123296493) [](ancestors = 123296493)